packages/terra-profile-image/src/avatar.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Let's run this SVG through https://jakearchibald.github.io/svgomg/ 
It will trim some of the unneeded comments / code while maintaining the same icon rendering.